gaios/pt-BR.lproj/Localizable.strings

```diff
@@ -1,438 +1,438 @@
-"id_send_all_requires_a_single" = "Send All requires a single recipient";
-"id_send_to" = "Send To";
-"id_sending" = "SENDING";
-"id_sending_request" = "Sending Request…";
-"id_sent_to" = "Sent to";
-"id_set_a_new_pin" = "Set a new PIN";
-"id_set_custom_fee_rate" = "Set custom fee rate";
-"id_set_twofactor_threshold" = "Set Two-Factor threshold";
-"id_set_up_pgp_key_for" = "Set up PGP key for communications";
-"id_set_up_twofactor_authentication" = "Set up Two-Factor authentication";
-"id_set_up_watchonly" = "Set up watch-only";
-"id_set_up_watchonly_credentials" = "Set up watch-only credentials for this wallet";
-"id_setting_updated" = "Setting updated";
-"id_settings" = "Settings";
-"id_setup_pin" = "Setup PIN";
-"id_setup_pin_and_s" = "Setup PIN and %@";
-"id_share" = "Share";
-"id_share_address" = "Share Address";
-"id_show_fiat" = "Show fiat";
-"id_signup_in_progress" = "Signup in progress";
-"id_simplified_payment_verification" = "Simplified Payment Verification verifies your transactions and balance using the Bitcoin network, outside of Green's servers. Enabling adds bandwidth and processing requirements";
-"id_skip" = "SKIP";
-"id_skip_not_recommended" = "Skip (Not Recommended)";
-"id_slide_to_send" = "Slide to Send";
-"id_slow" = "Slow";
+"id_pin_has_to_be_between_4_and_15" = "O PIN deve ter entre 4 e 15 dígitos";
+"id_pins_do_not_match_please_try" = "Os PINs não coincidem, por favor, tente novamente.";
+"id_please_disable_biometric" = "Por favor, desabilite a autenticação biométrica antes de remover o PIN";
+"id_please_enable_another" = "Por favor, habilite outro método de autenticação para evitar perder acesso aos seus fundos.";
+"id_please_enable_camera" = "Por favor, ative as permissões da câmera para utilizá-la.";
+"id_please_enable_pin" = "Por favor, ative a autenticação por PIN antes de adicionar ou remover a autenticação biométrica";
+"id_please_enter_the_passphrase_for" = "Por favor, digite a senha para sua carteira de hardware.";
+"id_please_follow_the_instructions" = "Por favor, siga as instruções em sua carteira de hardware.";
+"id_please_provide_your_1s" = "Por favor, forneça seu %1$@.";
+"id_please_provide_your_1s_code" = "Por favor, forneça seu código do %1$@.";
+"id_please_provide_your_passphrase" = "Por favor, forneça sua senha:";
+"id_please_reconnect_your_hardware" = "Por favor, reconecte sua carteira de hardware antes de tentar novamente.";
+"id_please_secure_your_mnemonic_and" = "Por favor, proteja seu mnemônico e confirme que concorda com os Termos de Serviço";
+"id_please_select_the_checkbox" = "Por favor, marque a caixa de marcação";
+"id_please_tap_card" = "Por favor, encoste o cartão";
+"id_privacy_policy" = "Política de Privacidade";
+"id_problem_with_key_1s" = "Problema com a chave %1$@";
+"id_providing_an_email_enables" = "Fornecer um email ativa notificações e a recuperação de transações.";
+"id_proxy_settings" = "CONFIGURAÇÕES DE PROXY";
+"id_qr_scanner" = "Escaner de QR";
```

Comment:
Escanear

Reply:
This one is "Escaner" I believe, the original in English is "QR Scanner"

Reply:
Yes. this is for scanner. So it could be 'Escâner' in pt-BR. slightly different in pt-PT.